src/components/accordion/accordion.style.ts

```diff
@@ -285,19 +285,19 @@ const StyledAccordionContent = styled.div<StyledAccordionContentProps>`
   padding-top: var(--spacing100);
   overflow: hidden;
 
-  ${({ disableContentPadding }) =>
-    disableContentPadding &&
-    css`
-      padding: 0;
-    `}
-
   ${({ variant }) =>
     variant === "subtle" &&
     css`
       margin-left: var(--spacing150);
       padding: var(--spacing100) var(--spacing200) var(--spacing300);
       border-left: 2px solid var(--colorsUtilityMajor100);
     `}
+
+  ${({ disableContentPadding }) =>
+    disableContentPadding &&
+    css`
+      padding: 0;
```

Comment:
**suggestion(non-blocking):** I see that we've used the Design Tokens elsewhere in this file to specify values. Might be nice to update this value too whilst we're moving this code block.

```suggestion
      padding: var(--spacing000);
```